src/Microsoft.Tye.Core/UserSecretsBuilder.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.IO;
+using System.Text.Json;
+
+namespace Microsoft.Tye
+{
+    internal static class UserSecretsBuilder
+    {
+        public static string GetSecretValue(string envSource)
+        {
+            var sep = '/';
+            var index = envSource.IndexOf(sep);
+            var secretId = envSource[1..index];
+            var secretName = envSource[(index + 1)..];
+            return GetSecretValue(secretId, secretName);
+        }
+
+        private static string GetSecretValue(string secretId, string secretName)
```

Comment:
This needs to be cross platform.

Reply:
Do you have to give some suggestions? Isn't there another place we do this correctly in the codebase today?

Reply:
https://github.com/dotnet/tye/blob/77e01e4b9167db457e720abe3504faebc3a35c6b/src/Microsoft.Tye.Hosting/DockerRunner.cs#L349

Reply:
I can lean on [this](https://github.com/dotnet/tye/blob/master/src/Microsoft.Tye.Hosting/DockerRunner.cs#L349) and possibly extract that to some public or internal static class.

Reply:
Yep extract it and share it